### DISCUSSION TITLE
Icon request: fa-eletrical-sensor

### DISCUSSION THREAD

**Discussion by Vido** [FortAwesome/Font-Awesome#21208] [Icon Requests]
I've been envolved in IoT projects. Often we need do represent a device on a dashboard or select a charts from a particular sensor. Currently, there is no icon to represent this. Sometimes we use `fa-bolt` or `fa-stethoscope`, but both are far from ideal. 

This is a example. I don't know from where it came from. If you search on Google images, you will find many of them.

![electrical_sensor](https://cloud.githubusercontent.com/assets/956388/9736788/a5bd44d4-5619-11e5-9fcc-3684f4d9ac6b.png)

Some issues asking the same kind of icon: #1394  #1399

**Comment by astralsteve** [comment]
+1
great idea, im also getting involved in the IoT and finding this to be VERY useful addition.

**Comment by tagliala** [comment]
@Vido are  #1394 #1399 duplicates of this one?

**Comment by Displee** [comment]
+1

**Comment by Vido** [comment]
@tagliala ,

#1394 , IMHO, this issue does not provide sufficient description
 #1399, is a similar idea, but with a very different icon

Thanks

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
